### PR TITLE
fix: fix organization link fix distribution box and add info box

### PR DESCRIPTION
**Pull request by zalborzi**
https://github.com/user-attachments/assets/b4dc293c-ba2b-4317-b84e-40a81c1e85c8

## Summary by Sourcery

This pull request fixes the organization link in the dataset metadata section and enhances the user interface by adding informational tooltips to various metadata fields. It also improves the styling of the distribution accordion and updates the 'Contact Point' heading for better clarity.

- **Bug Fixes**:
    - Fixed the organization link in the dataset metadata section.
- **Enhancements**:
    - Added informational tooltips to various dataset metadata fields, including creation date, modification date, catalogue, access rights, identifier, languages, distributions, and keywords.
    - Improved the styling of the distribution accordion, including rounded corners and hover effects.
    - Updated the 'Contact Point' heading to 'Contact Point(s)' for clarity.

### CODE REVIEW

File: src/app/datasets/[id]/DistributionAccordion.tsx [hunk 1]
```diff
@@ -83,78 +87,100 @@ const DistributionAccordion = ({
...
-                  <strong className="text-sm font-semibold">FileType:</strong>
-                  <span className="text-sm ml-2">
-                    {distribution.format?.label || "NA"}
+                <div className="flex items-center relative">
+                  <span className="group flex items-center">
+                    <FontAwesomeIcon
+                      icon={faFile}
+                      className="text-primary align-middle mr-2"
+                    />
+                    <strong className="text-sm font-semibold">FileType:</strong>
```
Comment (brunopacheco1) [inline src/app/datasets/[id]/DistributionAccordion.tsx:130]: important: please add space between words.

File: src/app/datasets/[id]/DistributionAccordion.tsx [hunk 2]
```diff
@@ -42,11 +43,14 @@ const DistributionAccordion = ({
   return (
     <div className="accordion flex w-full flex-col items-center justify-center">
```
Comment (brunopacheco1) [inline src/app/datasets/[id]/DistributionAccordion.tsx:44]: important: please add the same space you have between description and metadata, in between metadata and distributions.

The content is too packed right now.

Screenshot 2024-07-30 at 15 21 09

**Review by brunopacheco1 — CHANGES_REQUESTED** [review]

File: src/app/datasets/[id]/DistributionAccordion.tsx (hunk 1 above)
Comment (zalborzi) [reply src/app/datasets/[id]/DistributionAccordion.tsx:130]: Ouch!

File: src/app/datasets/[id]/DistributionAccordion.tsx (hunk 2 above)
Comment (zalborzi) [reply src/app/datasets/[id]/DistributionAccordion.tsx:44]: done

**Comment by brunopacheco1** [comment]
important: please make keywords boxes round again.

image

**Comment by brunopacheco1** [comment]
Important: please check console error:

image

**Comment by brunopacheco1** [comment]
suggestion: themes should also have a infobox.

image

**Comment by zalborzi** [comment]
> suggestion: description is the only metadata without information box. Although self-explanatory, the behaviour should be the same for all metadata, for consistency.
> 
> image

I was a bit hesitant to add for this one but lets do what as you said to have consistent page and then see if it is too much or not.

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**